resource/supported.go

```diff
@@ -309,6 +315,8 @@ func (a *AWS) RawResources(resType TerraformResourceType) (interface{}, error) {
 		return a.SecurityGroup()
 	case Subnet:
 		return a.subnets()
+	case Trail:
+		return a.trails()
```

Comment:
would be great if you could rename everywhere accordingly :+1:

Reply:
Yep yep; apologies for this i saw inconsistencies with things like KmsAliases and got turned around quite a bit.